xbmc/input/InputCodingTable.h

```diff
@@ -30,6 +30,26 @@ class IInputCodingTable
   virtual int GetType() { return TYPE_WORD_LIST; }
 
   virtual ~IInputCodingTable() {}
+  /*! \brief Called for the active keyboard layout when it's loaded, stick any initialization here
+
+      This won't be needed for most implementations so we don't set it =0 but provide a default
+      implementation.
+
+      \return true on success, false on any fatal errors
+  */
+  virtual bool Initialize() { return true; }
```

Comment:
See my comment below (leave the return value, get ride of CInputCodingTableBaiduPY::m_initialized)